examples/esm/esmfold-hpu.py

```diff
@@ -0,0 +1,73 @@
+# This script is based on https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/protein_folding.ipynb
+
+import time
+
+import habana_frameworks.torch.core as htcore
+import torch
+from transformers import AutoTokenizer, EsmForProteinFolding
+from transformers.models.esm.openfold_utils.feats import atom14_to_atom37
+from transformers.models.esm.openfold_utils.protein import Protein as OFProtein
+from transformers.models.esm.openfold_utils.protein import to_pdb
+
+from optimum.habana.transformers.modeling_utils import adapt_transformers_to_gaudi
+
+
+def convert_outputs_to_pdb(outputs):
+    final_atom_positions = atom14_to_atom37(outputs["positions"][-1], outputs)
+    outputs = {k: v.to("cpu").numpy() for k, v in outputs.items()}
+    final_atom_positions = final_atom_positions.cpu().numpy()
+    final_atom_mask = outputs["atom37_atom_exists"]
+    pdbs = []
+    for i in range(outputs["aatype"].shape[0]):
+        aa = outputs["aatype"][i]
+        pred_pos = final_atom_positions[i]
+        mask = final_atom_mask[i]
+        resid = outputs["residue_index"][i] + 1
+        pred = OFProtein(
+            aatype=aa,
+            atom_positions=pred_pos,
+            atom_mask=mask,
+            residue_index=resid,
+            b_factors=outputs["plddt"][i],
+            chain_index=outputs["chain_index"][i] if "chain_index" in outputs else None,
+        )
+        pdbs.append(to_pdb(pred))
+    return pdbs
+
+
+adapt_transformers_to_gaudi()
+
+steps = 4
+device = torch.device("hpu")
+
+# This is the sequence for human GNAT1, because I worked on it when
+# I was a postdoc and so everyone else has to learn to appreciate it too.
```

Comment:
```suggestion
# This is the sequence for human GNAT1.
```

Reply:
will be fixed.
Thanks a lot for your quick and valuable comments. Very appreciate!